.gitignore

```diff
@@ -1,6 +1,8 @@
 # Generated by Cargo
 # will have compiled files and executables
 /target/
+/out
+packages/docs/.vscode
```

Comment:
Do we need this?

Reply:
Next13 it's auto-generating this automatically, I could see how to disable this option. So, I just ignored it.